src/runtime/pkg/katautils/config.go

```diff
@@ -521,6 +522,13 @@ func (h hypervisor) guestHookPath() string {
 	return h.GuestHookPath
 }
 
+func (h hypervisor) guestHookTimeout() int32 {
+	if h.GuestHookTimeout <= 0 {
```

Comment:
AFAIC logically this should be `<0` instead of a `<=0` as zero-timeout has its own semantic now.